docs/section10/10-9.md

```diff
@@ -0,0 +1,51 @@
+# Airside systems – Rule 10-9  
+**Schema Version:** 0.0.12  
+**Mandatory Rule:** True     
+**Rule ID:** 10-9  
+ 
+**Rule Description:** Where multiple HVAC zones or residential spaces are combined into a single thermal block, baseline HVAC System Types 5 or 6 efficiencies shall be based on the cooling equipment capacity of a single floor.  
+
+**Rule Assertion:** B-RMR = expected value                                             
+**Appendix G Section:** Section G3.1.2.1  
+**Appendix G Section Reference:** None  
+**Data Lookup:** None   
+**Evaluation Context:** Evaluate each HeatingVentilationAirAconditioningSystem   
+
+**Applicability Checks:**  
+
+1. Applicable to baseline HVAC system types 5 or 6 that include more than one floor.  
+
+**Function Call:**  
+
+1. get_baseline_system_types()
+2. get_hvac_systems_5_6_serving_multiple_floors_b()
+
+**Manual Check:** None.  
+ 
+## Rule Logic:  
+**Applicability Check 1:**  
+- Get baseline hvac system type dictionary: `baseline_hvac_sys_type_ids_dict_b = get_baseline_system_types(B_RMR)`
+- Get dictionary with hvac system ids serving more than one floor as keys and the number of floors served as values: `hvac_sys_serve_more_than_one_flr_dict_b = get_hvac_systems_5_6_serving_multiple_floors_b(B_RMR)`
```

Comment:
just a small comment here. This rule only applies to System 5 or 6, so after line 27, if baseline system type is not equal to 5 or 6, then we should have a not applicable outcome and not do the rest.
same for line 28, if the baseline system is 5 or 6 but it is not serving multiple floors, then the outcome should be NA

Reply:
@supriyagoel , I modified it accordingly but based on previous conversations we are not explicitly creating NA outcomes.